wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -6,13 +6,18 @@
 {% block bodyclass %}page-editor create model-{{ content_type.model }}{% endblock %}
 
 {% block content %}
-
     <header class="merged tab-merged">
         {% explorer_breadcrumb parent_page include_self=1 %}
 
         <div class="row">
             <div class="left col9 header-title">
-                <h1>{% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span></h1>
+                <h1>
+                    {% icon "doc-empty-inverse" class_name="header-title-icon" %}
+                    {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span>
+                </h1>
+                {% if content_type.model_class.get_page_description %}
+                    <p>{{ content_type.model_class.get_page_description }}</p>
```

Comment:
Can we change this `p` to `div`? One might return a safe string from `get_page_description` with html structure. This allows you to display thumbnail/previews etc.